crates/re_data_store/src/entity_properties.rs

```diff
@@ -66,10 +66,14 @@ pub struct EntityProperties {
     /// `None` means backprojection is disabled.
     pub backproject_pinhole_ent_path: Option<EntityPath>,
 
-    /// Used to scale the resulting point cloud.
-    pub backproject_scale: EditableAutoValue<f32>,
+    /// How many depth units per world-space unit. e.g. 1000 for millimeters.
+    ///
+    /// This corresponds to [`re_log_types::component_types::Tensor::meter`].
+    pub backproject_depth_meter: EditableAutoValue<f32>,
```

Comment:
the doc comment wants me to name this `backproject_world_space_from_depth_factor` or something similar. But I also don't want to start the `meter`-discussion again ;)

Reply:
`depth_from_world_scale` is probably the most accuracte